groups/cvo2/data.tf

```diff
@@ -0,0 +1,62 @@
+data "aws_vpc" "vpc" {
+  tags = {
+    Name = "vpc-${var.aws_account}"
+  }
+}
+
+data "aws_subnet_ids" "storage" {
+  vpc_id = data.aws_vpc.vpc.id
+  filter {
+    name   = "tag:Name"
+    values = ["sub-storage-*"]
+  }
+}
+
+data "aws_route_table" "private" {
+  vpc_id = data.aws_vpc.vpc.id
+  filter {
+    name   = "tag:Name"
+    values = ["rtb-${var.account}-001"]
+  }
+}
+
+data "aws_route53_zone" "private_zone" {
+  name         = local.internal_fqdn
+  private_zone = true
+}
+
+data "vault_generic_secret" "account_ids" {
+  path = "aws-accounts/account-ids"
+}
+
+data "vault_generic_secret" "internal_cidrs" {
+  path = "aws-accounts/network/internal_cidr_ranges"
+}
+
+data "vault_generic_secret" "netapp_account" {
+  path = "applications/shared-services-${var.aws_region}/netapp/account"
+}
+
+data "vault_generic_secret" "netapp_connector" {
+  path = "applications/shared-services-${var.aws_region}/netapp/connector-details"
+}
+
+data "vault_generic_secret" "netapp_cvo" {
+  path = "applications/${var.aws_account}-${var.aws_region}/netapp/cvo-inputs"
+}
```

Comment:
These data lookups are still referencing the "old" Vault entries and so will be reliant on the same secrets. 

We should look to create a new set of Vault secrets for the new deployment so they can be managed / handled separately.